src/Winium.Desktop.Driver/CommandExecutors/SendKeysToActiveElementExecutor.cs

```diff
@@ -12,9 +16,9 @@ internal class SendKeysToActiveElementExecutor : CommandExecutorBase
 
         protected override string DoImpl()
         {
-            var text = string.Join(string.Empty, this.ExecutedCommand.Parameters["value"]);
+            var chars = this.ExecutedCommand.Parameters["value"].Select(x => Convert.ToChar(x.ToString()));
 
-            CruciatusFactory.Keyboard.SendText(text);
+            WiniumKeyboard.GetInstance().SendKeys(chars.ToArray());
```

Comment:
Why not make `GetInstance()` a property?

But, if we dig deeper, I think there should be no Instance property, nor there should be `GetInstance()`, because keyboard type is setup in `DesiredCapabiltites`, meaning that keyborad type is linked to concrete session. At the moment we do not support more than one session at smae time, but we can support it in future, so its better to make this issue future proof. Closest to session entity is `Automator`. `Automator` holds both `ActualCapabilties` (including keyborad type) for current session and launched `Application` process reference. Thats why I think current keyboard (`WiniumKeyborad`) property should be stored in `Automator` and this property should be used to execute commands. In this way we will link current keyborad to concerete session make it future proof.

----

Почему бы не заменить GetInstance на свойство Instance, будет лаконичнее?

Но, как мне кажется, на самом деле тут Instance не должен быть свойством, тут вообще Instance не должно быть, т.к. тип используемой клавиатуры задается через desiredCapabiltites, т.е. привязано к конкретной сессии. Сейчас, правда, мы не поддерживаем более одной параллельной сессии, но самое близкое что мы имеем к сесси — Automator, именно к нему мы пивязываем ActulaCapabilities и запускаемое приложение. Поэтому, т.к. тип клавиатуры связан с сессией, мне кажется, что у Automator должно быть свойство с текущей WiniumKeyboard, и вот оно должно использоваться в командах.